src/components/App.jsx

```diff
@@ -32,9 +33,9 @@ const ContactsApp = props => {
       <Layout monocolumn="true">
         {isMobile && (
           <BarCenter>
-            <Title>
-              <span className={'fil-path-title'}>Contacts</span>
-            </Title>
+            <MuiCozyTheme>
+              <Typography variant="h4">Contacts</Typography>
```

Comment:
Not in this PR, "Contacts" should be inside a `t()` call

Reply:
It should be read from the manifest, not from a translation call (since app's name can be changed on the fly by the registry for instance)

Reply:
@Crash-- Je ne comprends pas bien comment ça marche, comment accéder au `manifest` après modification de celui-ci par la `registry` ?
Et est-ce que `client.appMetadata.slug` ne suffirait pas ?